sympy/solvers/solveset.py

```diff
@@ -776,6 +762,12 @@ def solveset_complex(f, symbol):
         return result
 
 
+def _solve_complex_hyperbolic(f, symbol):
```

Comment:
Here `_solve_complex_hyperbolic` is exactly same as `_solve_real_trig`, I suggest removing `_solve_complex_hyperbolic` and renaming `_solve_real_trig` as `rewrite_to_exp` or maybe `solve_trig_hyperbolic`.